src/carma-1-tenth/check_distance_to_arrival.py

```diff
@@ -22,23 +22,23 @@ def check_distance_to_arrival(bag_dir):
         metadata_dict : dict = yaml.load(f, Loader=yaml.SafeLoader)["rosbag2_bagfile_information"]
     storage_id = metadata_dict['storage_identifier']
     # Goal and Ack topics
+    rviz_topic = '/goal_pose'
     goal_topic = '/incoming_mobility_operation'
     ack_topic = '/outgoing_mobility_operation'
     # Open bag
     reader, type_map = open_bagfile(bag_dir, topics=[goal_topic, ack_topic], storage_id=storage_id)
     # Gather number of messages on each topic
     topic_count_dict = {entry["topic_metadata"]["name"] : entry["message_count"] for entry in metadata_dict["topics_with_message_count"]}
     # Number of goal messages must equal number of acks
-    if topic_count_dict[goal_topic] != topic_count_dict[ack_topic]:
-        print("Number of goal messages (%d) does not equal number of ack messages (%d)".format(topic_count_dict[goal_topic], topic_count_dict[ack_topic]))
-        return np.array([np.inf])
+    if topic_count_dict[goal_topic] + topic_count_dict[rviz_topic] != topic_count_dict[ack_topic]:
+        print("Number of goal messages ({0}) does not equal one less than number of ack messages ({1})".format(topic_count_dict[goal_topic] + topic_count_dict[rviz_topic], topic_count_dict[ack_topic]))
```

Comment:
Based on this message it seems like the conditional above should be checking for equality with `topic_count_dict[ack_topic] - 1`

Reply:
Yes the error message was outdated, just fixed